build/Targets/VSL.Imports.targets

```diff
@@ -502,6 +502,36 @@
     </FilesToSign>
   </ItemGroup>
 
+  <Target Name="IncludeResourcesDllForSigning" AfterTargets="CopyFilesToOutputDirectory"> 
+    <ItemGroup> 
+      <!-- Handle the resources dll if there are any --> 
+      <FilesToSign Condition="'$(ShouldSignBuild)' == 'true' AND '$(NonShipping)' != 'true' AND '$(ProjectSystemLayer)' != 'Dependency'" Include="$(OutDir)\**\$(AssemblyName).resources.dll"> 
+          <Authenticode>Microsoft</Authenticode> 
+          <StrongName>$(StrongNameCertificateFriendlyId)</StrongName> 
+      </FilesToSign> 
+    </ItemGroup> 
+  </Target>
+
+  <!-- This target is similar to SatelliteDllsProjectOutputGroup except it includes path from the final output path rather than intermediate output path -->
+  <Target Name="SatelliteDllsProjectOutputGroupWithFinalOutputPath" 
```

Comment:
SatelliteDllsProjectOutputGroup already has metadata with FinalOutputPath:

https://github.com/Microsoft/msbuild/blob/f52cb1426fdca1d650c2018deb34ce50ccfd00b1/src/XMakeTasks/Microsoft.Common.CurrentVersion.targets#L5336

I think this can be simplified to 

<Target Name="SatelliteDllsProjectOutputGroupWithFinalOutputPath" DependsOnTargets="SatelliteDllsProjectOutputGroup"
Returns="@(SatelliteDllsProjectOutputGroupOutput->'%(FinalOutputPath)'" />

See https://github.com/NuGet/Home/issues/3880 where this is how it was fixed for pack